blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/AbstractBitbucketScm.java

```diff
@@ -361,7 +361,35 @@ protected StaplerRequest2 getStaplerRequest2(){
 
     @Restricted(NoExternalUse.class)
     public static @NonNull String normalizeApiUrl(@NonNull String apiUrl){
-        String serverUrl = BitbucketEndpointConfiguration.normalizeServerUrl(apiUrl);
+        try {
```

Comment:
This looks like it's the same as [URLUtils.normalizeURL](https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/9888e219b456f7bd4ee3128247d7513490e27edd/src/main/java/com/cloudbees/jenkins/plugins/bitbucket/impl/util/URLUtils.java#L61). Could we just use that instead?

Reply:
URLUtil is internal

Reply:
whoops, thanks for the clarification!